### PR TITLE
Bug 1957244 update Ingress configuration description for OpenShift API Server Operator

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1957244

Preview build: 
https://deploy-preview-32249--osdocs.netlify.app/openshift-enterprise/latest/networking/ingress-operator.html#nw-installation-ingress-config-asset_configuring-ingress

### CODE REVIEW

File: modules/nw-installation-ingress-config-asset.adoc [hunk 1]
```diff
@@ -20,14 +19,8 @@ spec:
...
+The installation program stores this asset in the `cluster-ingress-02-config.yml` file in the `manifests/` directory. This `Ingress` resource defines the cluster-wide configuration for Ingress. This Ingress configuration is used as follows:
 
-* The Ingress Operator uses the domain from the cluster Ingress configuration as
-the domain for the default Ingress Controller.
+* The Ingress Operator uses the domain from the cluster Ingress configuration as the domain for the default Ingress Controller.
 
-* The OpenShift API server operator uses the domain from the cluster Ingress
-configuration as the domain used when generating a default host for a `Route`
-resource that does not specify an explicit host.
+* The OpenShift API Server Operator uses the domain from the cluster Ingress configuration as the domain. This domain is also used when generating a default host for a `Route` resource that does not specify an explicit host.
```
Comment (bmcelvee) [inline modules/nw-installation-ingress-config-asset.adoc:26]: This is the only bullet with a content change. The rest of the changes are to remove hard returns.

Comment (bmcelvee) [reply modules/nw-installation-ingress-config-asset.adoc:26]: Old content: 
"The OpenShift API server operator uses the domain from the cluster Ingress configuration as the domain used when generating a default host for a `Route` resource that does not specify an explicit host."

New content: 
"The OpenShift API Server Operator uses the domain from the cluster Ingress configuration as the domain. This domain is also used when generating a default host for a `Route` resource that does not specify an explicit host."

File: modules/nw-installation-ingress-config-asset.adoc [hunk 2]
```diff
@@ -20,14 +19,8 @@ spec:
...
+The installation program stores this asset in the `cluster-ingress-02-config.yml` file in the `manifests/` directory. This `Ingress` resource defines the cluster-wide configuration for Ingress. This Ingress configuration is used as follows:
 
-* The Ingress Operator uses the domain from the cluster Ingress configuration as
-the domain for the default Ingress Controller.
+* The Ingress Operator uses the domain from the cluster Ingress configuration as the domain for the default Ingress Controller.
 
-* The OpenShift API server operator uses the domain from the cluster Ingress
-configuration as the domain used when generating a default host for a `Route`
-resource that does not specify an explicit host.
+* The OpenShift API Server Operator uses the domain from the cluster Ingress configuration. This domain is also used when generating a default host for a `Route` resource that does not specify an explicit host.
```
Comment (bmcelvee) [inline modules/nw-installation-ingress-config-asset.adoc:26]: This is the only bullet with a content change. The rest of the changes are to remove hard returns.

Old content:
"The OpenShift API server operator uses the domain from the cluster Ingress configuration as the domain used when generating a default host for a Route resource that does not specify an explicit host."

New content:
"The OpenShift API Server Operator uses the domain from the cluster Ingress configuration. This domain is also used when generating a default host for a `Route` resource that does not specify an explicit host."

**Comment by bmcelvee** [comment]
QE approved this change, see BZ for details.

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**